src/ray/raylet/node_manager.h

```diff
@@ -53,26 +54,33 @@ class NodeManager {
   void ProcessNodeManagerMessage(std::shared_ptr<TcpClientConnection> node_manager_client,
                                  int64_t message_type, const uint8_t *message);
 
-  void ClientAdded(gcs::AsyncGcsClient *client, const UniqueID &id,
-                   const ClientTableDataT &data);
+  ray::Status RegisterGcs();
 
   void HeartbeatAdded(gcs::AsyncGcsClient *client, const ClientID &id,
                       const HeartbeatTableDataT &data);
 
  private:
+  // Handler for the addition of a new GCS client.
+  void ClientAdded(gcs::AsyncGcsClient *client, const UniqueID &id,
```

Comment:
I think we should document the arguments for these methods. In particular, I'm not sure what `id` is (part of the reason is that it is a `UniqueID` as opposed to something more specific).

Reply:
Good point, thanks. In this particular case, I don't think `UniqueID` should even be an argument here...I'll remove it.